item-api/src/main/java/com/example/itemapi/domain/manager/ItemManager.java

```diff
@@ -0,0 +1,34 @@
+package com.example.itemapi.domain.manager;
+
+import java.math.BigDecimal;
+
+import org.springframework.stereotype.Component;
+
+import com.example.itemapi.domain.model.Item;
+import com.example.itemapi.domain.model.Price;
+import com.example.itemapi.domain.repository.ItemRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Component
+public class ItemManager {
+
+    private final ItemRepository itemRepository;
+
+    public void addItem(String name, int price, int stock) {
```

Comment:
여기선 price 를 인자로 int 로 받고있네요 😎
BigDecimal 로 수정하거나 String 으로 받고  BigDecimal 로 변환해 사용하는 것도 좋아보입니다!


```suggestion
    public void addItem(String name, BigDecimal  price, int stock) {
```